subprojects/launcher/src/integTest/groovy/org/gradle/launcher/BuildEnvironmentIntegrationTest.groovy

```diff
@@ -227,8 +227,8 @@ task check {
     }
 
     @Issue("https://github.com/gradle/gradle/issues/1001")
+    @ToBeFixedForConfigurationCache(because = "Init scripts don't have access to providers yet")
```

Comment:
@eskatos FYI, a missing feature I think

Reply:
Indeed! I just opened https://github.com/gradle/gradle/issues/16849 for this